pithos/pithos.py

```diff
@@ -22,13 +22,24 @@
 import signal
 
 import gi
-gi.require_version('Gst', '1.0')
 from gi.repository import Gst, GstPbutils, GObject, Gtk, Gdk, Pango, GdkPixbuf, Gio, GLib
 import contextlib
 import html
 import math
 import urllib.request, urllib.error, urllib.parse
 import json
+
+try:
+    gi.require_version('Gst', '1.0')
+except ValueError, e:
+    if e[0].startswith("Namespace Gst not available for version"):
+        print "Received error %s from Gstreamer." % e[0]
```

Comment:
>The app itself should check for missing libs at runtime

Well uh, yea it does, you get that for free. Plugins handle it nicely and pithos refuses to start basically, as expected.